main/HPSF/Variant.cs

```diff
@@ -512,11 +514,13 @@ public static String GetVariantName(long variantType)
         public static int GetVariantLength(long variantType)
         {
             long key = (int)variantType;
-            if (numberToLength.Contains(key))
```

Comment:
compared this to Java version and there was actually a bug here, it should have been not contains -> -2